.github/workflows/test.yml

```diff
@@ -25,8 +25,22 @@ jobs:
       - name: Install
         run: npm ci
 
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+
       - name: Test and generate code coverage info
         run: npm test
 
+      - name: Test Playwright and generate test report
+        run: npm run test:playwright-ct:all
+
       - name: Upload code coverage info to Coveralls
         uses: coverallsapp/github-action@v2
+
+      - name: Upload Playwright test report
```

Comment:
You can download artifact and run the report locally as you would on local machine. If we want to generate some report, create issue for that. Currently, there is no official plugin for this. I just wanted to point this out.